src/hooks/useExternalWalletSectionsData.ts

```diff
@@ -2,23 +2,19 @@ import { useMemo } from 'react';
 import { AssetListType } from '../components/asset-list/RecyclerAssetList2';
 import useFetchHiddenTokens from './useFetchHiddenTokens';
 import useFetchShowcaseTokens from './useFetchShowcaseTokens';
-import useFetchUniqueTokens from './useFetchUniqueTokens';
 import { buildBriefUniqueTokenList } from '@/helpers/assets';
+import { useLegacyNFTs } from '@/resources/nfts';
 
 export default function useExternalWalletSectionsData({
   address,
-  infinite = false,
   type,
 }: {
   address?: string;
-  infinite?: boolean;
   type?: AssetListType;
 }) {
-  const {
-    data: uniqueTokens,
-    isLoading: isUniqueTokensLoading,
-    isSuccess: isUniqueTokensSuccess,
-  } = useFetchUniqueTokens({ address, infinite });
+  const { data: uniqueTokens, isInitialLoading } = useLegacyNFTs({
+    address: address ?? '',
+  });
```

Comment:
What happens if we get an error here? Just curious, if it's too much to add in this PR ignore, but could be something to think about while we're here.

Reply:
nothing special. we don't have communicate errors w/ loading NFTs in the UI, I think thats something good to think about though